scripts/generic-ci-tests.sh

```diff
@@ -85,7 +85,9 @@ case "$TEST_SUITE" in
         echo "Running code complexity report (python)."
         paver run_complexity > reports/code_complexity.log || echo "Unable to calculate code complexity. Ignoring error."
         echo "Running safe template linter report."
-        paver run_safelint -l $SAFELINT_THRESHOLD > safelint.log || { cat safelint.log; EXIT=1; }
+        paver run_safelint -t $SAFELINT_THRESHOLDS > safelint.log || { cat safelint.log; EXIT=1; }
+        echo "Running safe commit linter report."
+        paver run_safecommit_report > safecommit.log || { cat safecommit.log; EXIT=1; }
```

Comment:
@benpatterson The output is going to the console when running from command line, using `tee` inside `run_safecommit_report`.  However, it is not currently going to the Jenkins console.  The only one that is doing that is run_quality below which does not redirect to a log file.

Let's discuss what you want and how I should best implement it.  Thanks!